### ISSUE TITLE
Decreased KCO

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6498]
**Preferred term label:**
Decreased KCO
**Synonyms**
**Definition (free text, please give PubMed ID)**
Decreased diffusion capacity of the lung for carbon monoxide (CO) rate constant (efficiency index)PMID: 11334115
**Parent term (use hpo.jax.org/app)**
Abnormal KCO
**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Diffuse alveolar damage
Combined pulmonary fibrosis-emphysema syndrome ORPHA:300564
Idiopathic pulmonary fibrosis ORPHA:2032 
Sarcoidosis ORPHA:797 
Asbestosis ORPHA:2302
Vasculitis ORPHA:52759
Chronic thromboembolic pulmonary hypertension ORPHA:70591 
**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
Is KCO different from DLCO?
https://hpo.jax.org/app/browse/term/HP:0045049

**Comment by eswietlik** [comment]
> Is KCO different from DLCO?
> https://hpo.jax.org/app/browse/term/HP:0045049

DLCO is the product of KCO and VA, I would replace DLCO with KCO and VA, but happy to discuss.

**Comment by pnrobinson** [comment]
addeded
